kafka-source/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>kafka-source</artifactId>
+    <packaging>jar</packaging>
+    <name>kafka-source</name>
+    <description>Kafka Source stream module</description>
+
+    <parent>
+        <groupId>org.springframework.cloud.stream.module</groupId>
+        <artifactId>spring-cloud-stream-modules</artifactId>
+        <version>1.0.0.BUILD-SNAPSHOT</version>
+    </parent>
+
+    <properties>
+        <start-class>org.springframework.cloud.stream.module.kafka.source.KafkaSourceApplication</start-class>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.cloud.stream.module</groupId>
+            <artifactId>spring-cloud-stream-modules-common-configuration</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.integration</groupId>
+            <artifactId>spring-integration-java-dsl</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.integration</groupId>
+            <artifactId>spring-integration-kafka</artifactId>
+            <version>1.3.0.RELEASE</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>log4j</groupId>
+                    <artifactId>log4j</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>slf4j-log4j12</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka_2.10</artifactId>
+            <classifier>test</classifier>
+            <version>0.8.2.2</version>
+            <optional>true</optional>
+            <exclusions>
+                <exclusion>
+                    <groupId>log4j</groupId>
+                    <artifactId>log4j</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>slf4j-log4j12</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.cloud.stream.module</groupId>
+            <artifactId>spring-cloud-stream-modules-test-support</artifactId>
+            <version>1.0.0.BUILD-SNAPSHOT</version>
```

Comment:
The `version` is redundant here.
We are in the same project with its modules.

I wonder if we even can avoid explicit `1.0.0.BUILD-SNAPSHOT` in the `<parent>` definition.
For example use some external `properties`, like we have with Gradle...